src/main/kotlin/no/nav/familie/ef/mottak/config/Konstanter.kt

```diff
@@ -1,6 +1,6 @@
 package no.nav.familie.ef.mottak.config
 
 const val DOKUMENTTYPE_OVERGANGSSTØNAD = "OVERGANGSSTØNAD_SØKNAD"
+const val DOKUMENTTYPE_BARNETILSYN = "BARNETILSYN_SØKNAD"
 const val DOKUMENTTYPE_SKJEMA_ARBEIDSSØKER = "SKJEMA_ARBEIDSSØKER"
-const val DOKUMENTTYPE_VEDLEGG = "OVERGANGSSTØNAD_SØKNAD_VEDLEGG"
-
+const val DOKUMENTTYPE_VEDLEGG = "SØKNAD_VEDLEGG"
```

Comment:
Er denne grei, eller trenger vi en dokumenttype per vedlegg per søknad?

Reply:
I neste versjon av innsending er det meningen at man skal kunne sende inn flere søknader på en gang. Det blir vanskelig å holde orden på hvilke vedlegg som skal hvor. Noen skal bare til den ene, noen skal til begge. Vi tenker derfor det er greit at vi sender alle vedlegg med alle søknader. Saksbehandler kan vurdere om et vedleg er relevant.

Reply:
Godt poeng, @ma10s - noen vedlegg vil jo være aktuelle for flere søknader i så måte.

Reply:
Ups - slettet min kommentar - uhell. Sorry! Skulle editere. 
Det jeg slettet: 
Senere sender vi inn flere søknader i samme submit. I mottak må vi da legge alle vedlegg til alle søknader. Noen vedlegg er påkrevd i flere søknader. Enkel løsning => vi sender inn alle vedlegg til alle søknader. Saksbehandler vurderer om vedlegg er relevant.

Reply:
Så lenge dokumenttypen finnes i integrasjoner: 
https://github.com/navikt/familie-integrasjoner/pull/154/files

Reply:
Hva skal den fila i integrasjoner brukes til?
Hvis denne typen er noe vi skal bruke senere så tenker jeg att jag ikke skal endre navn på den ettersom vi allerede har arkivert vedleggen.

Reply:
Jeg tror ikke dokumenttypen brukes til annet enn mapping av metadata for dokumenttype i integrasjoner. 

@bent-lorentzen - stemmer dette? 

Her er overgangsstønad - vedlegg fra integrasjoner: 
@Component
object OvergangsstønadVedleggMetadata : DokumentMetadata {

    override val journalpostType: JournalpostType = JournalpostType.INNGAAENDE
    override val fagsakSystem: String? = null
    override val tema: String = "ENF"
    override val behandlingstema: String? = null // https://confluence.adeo.no/display/BOA/Behandlingstema
    override val kanal: String? = null
    override val dokumentTypeId: String = "OVERGANGSSTØNAD_SØKNAD_VEDLEGG"
    override val tittel: String? = null
    override val brevkode: String? = null
    override val dokumentKategori: String = "IS"
} 


Metadata hentes ut f.eks slik: 
 fun getMetadata(deprecatedDokument: Dokument): DokumentMetadata {
        return metadata[deprecatedDokument.dokumentType] ?: error("Ukjent dokumenttype ${deprecatedDokument.dokumentType}")
    }

Reply:
Fikset i https://github.com/navikt/familie-ef-mottak/pull/232/commits/4ec31d61e60fd84786094e856b4d2800824eace5